### PR TITLE
chore: Update licenses checksums for git recipes

### PR DESCRIPTION
**Pull request by lluiscampos**
Following up from the copyright year update in the license files

See:
* https://github.com/mendersoftware/mender-artifact/pull/576
* https://github.com/mendersoftware/mender-configure-module/pull/105
* https://github.com/mendersoftware/mender-connect/pull/126
* https://github.com/mendersoftware/mender-flash/pull/10
* https://github.com/mendersoftware/mender-snapshot/pull/18

### CODE REVIEW

**Comment by lluiscampos** [comment]
@kacf @oleorhagen This one cannot be tested in CI because `grub-mender-grubenv` and `app-update-module` repos don't have the logic in mender-qa to build the git recipes.

File: meta-mender-core/recipes-bsp/grub-mender-grubenv/grub-mender-grubenv_git.bb [hunk 1]
```diff
@@ -6,4 +6,4 @@ SRCREV = "${GRUB_MENDER_GRUBENV_REV}"
 PV = "1.3.0+git${SRCREV}"
 
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=b4b4cfdaea6d61aa5793b92efd42e081"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=30b4554c64108561c0cb1c57e8a044f0"
```
Comment (kacf) [inline meta-mender-core/recipes-bsp/grub-mender-grubenv/grub-mender-grubenv_git.bb:9]: This one shouldn't be updated unless you also update `GRUB_MENDER_GRUBENV_REV` (which isn't necessary unless something else has changed too, IMHO).

File: meta-mender-core/recipes-update-modules/mender-app-update-module/mender-app-update-module_git.bb [hunk 2]
```diff
@@ -2,7 +2,7 @@ DESCRIPTION = "Mender Application Update Module for supporting containerized app
 
 HOMEPAGE = "https://mender.io"
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=b4b4cfdaea6d61aa5793b92efd42e081"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=30b4554c64108561c0cb1c57e8a044f0"
```
Comment (kacf) [inline meta-mender-core/recipes-update-modules/mender-app-update-module/mender-app-update-module_git.bb:5]: This one also should not be updated without updating `SRCREV`. The two go hand in hand.

**Review by kacf — CHANGES_REQUESTED** [review]

File: meta-mender-core/recipes-update-modules/mender-app-update-module/mender-app-update-module_git.bb (hunk 2 above)
Comment (lluiscampos) [reply meta-mender-core/recipes-update-modules/mender-app-update-module/mender-app-update-module_git.bb:5]: Thanks. I got it now.

If I drop these two I can actually run the pipeline too

**Review by kacf — APPROVED** [review]

**Pull request merged**